lib/kernel.coffee

```diff
@@ -11,20 +11,18 @@ WatchSidebar = require './watch-sidebar'
 
 module.exports =
 class Kernel
-    constructor: (@kernelSpec, @grammar, @config, @configPath, @onlyConnect = false) ->
+    constructor: (@onDestroy, @kernelName, @kernelSpec, filePath, @config, @configPath, @onlyConnect = false) ->
```

Comment:
Actually, nothing outside the constructor needs it.

I think I'll keep it constructor-local, and fix the misreference below.